DEPLOYMENT.md

```diff
@@ -23,10 +23,11 @@ Rename it to `license.json` and copy it into the root folder of this project.
 
 Us the [Curity developer portal](https://developer.curity.io/releases/token-handler) to download one of the OAuth Proxy zip files to the root folder of this project:
 
-- [Kong OAuth Proxy 2.0+](https://developer.curity.io/releases/token-handler?proxy=kong)
-- [OpenResty OAuth Proxy 2.0+](https://developer.curity.io/releases/token-handler?proxy=openresty)
+- [Kong OAuth Proxy 2.0.0+](https://developer.curity.io/releases/token-handler?proxy=kong)
+- [OpenResty OAuth Proxy 2.0.1+](https://developer.curity.io/releases/token-handler?proxy=openresty)
+- [NGINX OAuth Proxy 2.0.0+](https://developer.curity.io/releases/token-handler?proxy=nginx)
```

Comment:
Add `nginx` to the possible options for `OAUTH_PROXY_TYPE`

Reply:
done